### PR TITLE
Feature/validation flag [LEI-231]

### PR DESCRIPTION
**Pull request by samanehsan**
Refs: https://openscience.atlassian.net/browse/LEI-231
Companion to: https://github.com/CenterForOpenScience/isp/pull/76

## Purpose
Add the ability to turn validation off for testing.

## Summary of changes
If the `validate` flag is set to false, allow users to continue through the survey without having to answer every question.

### CODE REVIEW

File: exp-player/addon/components/exp-card-sort/component.js [hunk 1]
```diff
@@ -168,6 +169,12 @@ export default ExpFrameBaseComponent.extend({
         }
         return responses;
     }).volatile(),
+
+    allowNext: Ember.computed('cards.[]', function() {
+        console.log(this.get('cards').length);
```
Comment (abought) [inline exp-player/addon/components/exp-card-sort/component.js]: Please remove `console.log` statements before committing

Also, I notice the indentation is different between these two lines?

File: exp-player/addon/components/exp-card-sort/component.js [hunk 2]
```diff
@@ -168,6 +169,12 @@ export default ExpFrameBaseComponent.extend({
...
+      return this.get('cards').length === 0 || !config.validate;
```
Comment (abought) [inline exp-player/addon/components/exp-card-sort/component.js]: Should this depend on two conditions both being met? Using "or", it seems that there is a situation where only the first condition would need to be satisfied for this to return true.

(In other words, the `config.validate` flag might be ignored)

Comment (samanehsan) [inline exp-player/addon/components/exp-card-sort/component.js]: This should return true if there are no cards left to sort, or if validation is turned off. I changed it to explicitly check `config.validate` for clarity.

**Comment by samanehsan** [comment]
Done responding @abought! ☔

**Pull request merged**